.github/workflows/deploy-infra.yaml

```diff
@@ -106,7 +104,7 @@ jobs:
       max-parallel: 1
       matrix:
         environment-type: ${{ fromJson(needs.setup.outputs.matrix) }}
-        environment-name: [Biomage, HMS]
+        environment-name: [HMS]
         exclude:
           - environment-name: HMS
             environment-type: staging
```

Comment:
same as the previous comment in `deploy-changed-cf.yaml`. We should be able to remove this right?